### PR TITLE
chore: improve phone number init signup param doc

### PR DESCRIPTION
**Pull request by jkdowdle**
## Motivation

Make it more clear the format in which the phone number should be provided

### CODE REVIEW

**Review by swain — COMMENTED** [review]
I think the commit looks good, but I think you targeted the wrong branch on this PR

**Review by aecorredor — COMMENTED** [review]
+1 on the wrong target branch

File: src/APIBasedAuth.ts [hunk 1]
```diff
@@ -373,6 +393,38 @@ declare namespace APIBasedAuth {
...
+    /**
+     * Password for the user. In the case that a password is not
+     * provided it is assumed that this is a passwordless user and
+     * a password is auto generated for them on the server. The
+     * user is still able to go through the forgot password flow
+     * to set it at a later time
+     */
+    password?: string;
+    /**
+     * An eleven digit phone number for the user
```
Comment (rob-snyder) [inline src/APIBasedAuth.ts:421]: Super minor, but the phone number format is E.164 structure, if you want to reference it.

Comment (jkdowdle) [reply src/APIBasedAuth.ts:421]: Awesome! Thank you

**Review by rob-snyder — APPROVED** [review]

**Review by aecorredor — APPROVED** [review]

**Pull request merged**